### ISSUE TITLE
Fix number of epoch train for all splits

### ISSUE THREAD

**Issue by alpoler** [KevinMusgrave/powerful-benchmarker#74]
I want to train all splits in determined number of epoch to make comparision with other trainings. Are there any way to take epoch number as constant for all splits ?

**Comment by KevinMusgrave** [comment]
Do this:
```
--patience null --num_epochs_train 100
```
Then it'll train for 100 epochs regardless of accuracy. (It won't stop if the accuracy plateaus.) 

When you evaluate, it will still use the best performing model.
